src/components/block/DonationProjectCard/DonationProjectCard.module.css

```
@@ -0,0 +1,33 @@
+.container {
+  height: 240px;
+  position: relative;
+}
+
+.bottombox {
+  bottom: 0;
+  display: flex;
+  flex-direction: row;
+  height: 72px;
+  position: absolute;
+  width: inherit;
+}
+
+.bar {
+  background: var(--color-sheltertech-blue);
+  height: 100%;
+  margin-right: 12px;
+  width: 8px;
+}
+
+.title {
+  color: var(--color-white);
+  font-family: var(--font-family-lato);
+  font-size: var(--font-size-subtitle);
+  font-weight: bold;
```

One of the things that set up in https://github.com/ShelterTechSF/sheltertech.org/pull/63 is being able to set multiple of these properties at the same time with the [`font`](https://developer.mozilla.org/en-US/docs/Web/CSS/font) shorthand property.

```suggestion
  font: var(--font-subtitle);
```

---

👍 Changed.